packages/babel-standalone/src/preset-es2015.js

```diff
@@ -35,19 +34,6 @@ export default (_, opts) => {
     if (opts.spec !== undefined) spec = opts.spec;
   }
 
-  if (typeof loose !== "boolean") {
```

Comment:
What happened with this?

Reply:
Oh hmm I think I ended up removing this since it's for standalone and it would be a UI option in the repl, but we can add it back too.